crates/rpc/src/handlers/committee.rs

```diff
@@ -0,0 +1,84 @@
+use std::vec;
+
+use actix_web::{
+    HttpResponse, Responder, get,
+    web::{Data, Path, Query},
+};
+use ream_consensus::{constants::SLOTS_PER_EPOCH, misc::compute_start_slot_at_epoch};
+use ream_storage::db::ReamDB;
+use serde::Serialize;
+
+use super::state::get_state_from_id;
+use crate::types::{
+    errors::ApiError,
+    id::ID,
+    query::{EpochQuery, IndexQuery, SlotQuery},
+    response::BeaconResponse,
+};
+
+#[derive(Debug, Serialize, Clone)]
+pub struct CommitteeData {
+    #[serde(with = "serde_utils::quoted_u64")]
+    pub index: u64,
+    #[serde(with = "serde_utils::quoted_u64")]
+    pub slot: u64,
+    #[serde(with = "serde_utils::quoted_u64_vec")]
+    pub validators: Vec<u64>,
+}
+
+impl CommitteeData {
+    pub fn new(index: u64, slot: u64, validators: Vec<u64>) -> Self {
+        Self {
+            index,
+            slot,
+            validators,
+        }
+    }
+}
+
+/// Called by `/states/<state_id>/committees` to get the Sync Committee Data of state.
```

Comment:
```suggestion
/// Called by `/states/<state_id>/committees` to get the Committee Data of state.
```

Is this function specific to sync committees or comittees in general?

Reply:
It is for committees in general

Reply:
We should remove "Sync " then